interfaces/builtin/camera.go

```diff
@@ -49,11 +52,14 @@ const cameraConnectedPlugAppArmor = `
 /run/udev/data/+usb:* r,
 /sys/class/video4linux/ r,
 /sys/devices/pci**/usb*/**/video4linux/** r,
+/sys/devices/pci**/**/**/**/**/video4linux/** r,
+/sys/devices/virtual/video4linux/** r,
```

Comment:
Note that `**` matches slashes.  Are we specifically trying to match paths with a minimum number of slashes here?  Or would it make sense to simplify and merge the three rules into one?

    /sys/devices/**/video4linux/** r,

Reply:
Good point. I've merged the three rules to one.